_source/user-guide/kibana/index.md

```diff
@@ -8,24 +8,25 @@ tags:
   - kibana
 contributors:
   - imnotashrimp
+  - shalper
 ---
 
 When you first sign in to Logz.io, you'll find yourself at the Discover page. You can use the Discover page to filter recent logs, search for specific events, and create alerts based on your search queries.
 
-![Kibana UI in Logz.io](https://dytvr9ot2sszz.cloudfront.net/logz-docs/kibana/kibana--discover-annotated.png)
+![Kibana UI in Logz.io](https://dytvr9ot2sszz.cloudfront.net/logz-docs/kibana/kibana--discover-annotated1.png)
 
 Query bar
-: This is where you type your Elasticsearch query
+: This is where you type your search query
 
-Field and account selection
-: Choose the accounts to search and the fields that will get a table column
+Account and field selection
+: Choose the accounts to search and add fields to the preview table
 
 Histogram
-: Time-limited view of your search query
-
-Cognitive Insights
-: If Logz.io notices patterns or anomalies in your logs, it will notify you with Cognitive Insights
+: This shows you the volume of data returned for your search query. You can select a time period directly from here
 
 Document table
-: A text view of the results of your search query
-{:.letter-labels}
+: View the results of your search query. Click on any log document to open it and view the fields and values
+
```

Comment:
There is a lot to add on the table and what's the existing functionality that is available there.